src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+public class ColorSupplier extends Lottery {
+
     public String getRandomColor() {
-        return null;
+
+        int randomIndex = getRandom().nextInt(Color.values().length);
+
+        return String.valueOf(Color.values()[randomIndex]);
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants